pkgs/top-level/all-packages.nix

```
@@ -35363,6 +35363,10 @@ with pkgs;
     branch = "canary";
   };
 
+  discord-development = callPackage ../applications/networking/instant-messengers/discord {
+    branch = "development";
+  };
+
```

```suggestion
  discord-development = import ../applications/networking/instant-messengers/discord {
    inherit lib stdenv;
    inherit (pkgs) callPackage fetchurl;
    branch = "development";
  };

```

This is an oversight caused by 64938ed, using `callPackage` here isn't necessary since internally it calls `callPackage` already and it breaks overriding, see #179906

---

Attempting to override as it is now overrides this [helper file](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/applications/networking/instant-messengers/discord/default.nix) instead of the [actual derivation](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/applications/networking/instant-messengers/discord/linux.nix) for Discord itself.

---

Ah, makes sense, thanks.